specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -27,6 +27,34 @@ interface TextOperations {
     },
     AnalyzeTextResult
   >;
+
+  @added(ContentSafety.Versions.v2023_10_15_Preview)
+  @summary("Analyze Text Jailbreak")
+  @doc("A synchronous API for the analysis of text jailbreak.")
+  @route("/text:detectJailbreak")
+  @post
+  detectTextJailbreak is Azure.Core.RpcOperation<
+    {
+      @body
+      @doc("The text jailbreak analysis request.")
+      body: AnalyzeTextJailbreakOptions;
```

Comment:
for SDK, should you follow GAed APIs and project this `body` to `options` for some languages? Same below.

Reply:
Added